x-pack/plugins/ml/public/application/components/validate_job/validate_job_view.js

```diff
@@ -208,7 +208,9 @@ export class ValidateJobUI extends Component {
     const duration = typeof getDuration === 'function' ? getDuration() : undefined;
     const fields = this.props.fields;
 
-    if (typeof job === 'object') {
+    // Run job validation only if a job config has been passed on and the duration makes sense to run it.
+    // Otherwise we skip the call and display a generic warning, but let the user move on to the next wizard step.
+    if (typeof job === 'object' && duration.start !== null && duration.end !== null) {
```

Comment:
nit, this whole condition could be wrapped in an outer `if (typeof job === 'object') {`

Reply:
Fixed in b366241. Just became aware that this is still plain JavaScript so I added another check against `typeof duration === 'object')` too.